CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+CHANGES IN VERSION 1.4
+======================================
+
+#### General
+- fix for reset script
+- fix for issues with cookie encoding
+- updated sample code
+- updated documentation
+- updated test suite
+
+#### iOS
+- Support for iOS 8.3
+- fix for issues relating to finding xcode folder
+- new capability `instruments` to specify custom path to instruments commandLine tool
```

Comment:
i would advertise this as a server flag `--instruments <path>` rather than a capability